docs/user/security/securing-communications/index.asciidoc

```diff
@@ -4,127 +4,184 @@
+server.ssl.enabled: true
+--------------------------------------------------------------------------------
+--
+
+. *Restart {kib}.*
+
+After making these changes, you must always access {kib} via HTTPS. For example, https://<your_kibana_host>.com.
 
 [[configuring-tls-kib-es]]
 ==== Encrypting traffic between {kib} and {es}
 
-NOTE: To perform this step, you must 
-{ref}/configuring-security.html[enable the {es} {security-features}] or you 
-must have a proxy that provides an HTTPS endpoint for {es}. 
+NOTE: To perform this step, you must {ref}/configuring-security.html[enable the {es} {security-features}] or you must have a proxy that
+provides an HTTPS endpoint for {es}.
+
+. *{ref}/configuring-tls.html#tls-http[Enable TLS on the HTTP layer in {es}].*
 
-. Specify the HTTPS URL in the `elasticsearch.hosts` setting in the {kib}
-configuration file, `kibana.yml`:
+. *Obtain the certificate authority (CA) certificate chain for {es}.*
 +
 --
-[source,yaml]
+{kib} needs the appropriate CA certificate chain to properly establish trust when connecting to {es}.
+
+If you followed the {es} documentation for {ref}/configuring-tls.html#node-certificates[Generating Node Certificates], then you likely have
+a PKCS#12 file for each your {es} nodes. You can extract the CA certificate chain from one of these files. For example:
+
+[source,sh]
 --------------------------------------------------------------------------------
-elasticsearch.hosts: ["https://<your_elasticsearch_host>.com:9200"]
+openssl pkcs12 -in elastic-certificates.p12 -cacerts -nokeys -out elasticsearch-ca.crt
 --------------------------------------------------------------------------------
 
-Using the HTTPS protocol results in a default
-`elasticsearch.ssl.verificationMode` option of `full`, which utilizes hostname
-verification.
+This will produce a PEM-formatted file named `elasticsearch-ca.crt` that contains all CA certificates from the PKCS#12 file.
+--
+
+. *Configure {kib} to trust the {es} server certificate(s) for the HTTP layer.*
++
+You need to specify the information required to access your {es} CA certificate chain.
+
+.. If your CA certificate chain is in PEM format:
++
+--
+Specify one or more CA certificates in `kibana.yml`:
 
-For more information, see <<settings,{kib} configuration settings>>. 
+[source,yaml]
+--------------------------------------------------------------------------------
+elasticsearch.ssl.certificateAuthorities: ["/path/to/elasticsearch-ca.crt"]
+--------------------------------------------------------------------------------
 --
 
-. Specify the {es} cluster's CA certificate chain in `kibana.yml`:
+
+.. If your CA certificate chain is in a PKCS#12 file:
 +
 --
+WARNING: You should not use a PKCS#12 file that contains a private key. This is an unnecessary security risk. If you only have a PKCS#12
+file that contains a private key, a safer approach is to extract the CA certificate chain in PEM format as described above.
```

Comment:
I am afraid that some users that are following all of the setup instructions will attempt to use `elastic-stack-ca.p12` as a trust store. However, I think we still need to mention that using a PKCS#12 trust store is an option here, otherwise users may not be aware of it.

This warning is the best compromise I could come up with to discourage insecure practices. The warning is sort of vague, but I think it gets the point across.